src/i18n/locales/en.json

```diff
@@ -31,7 +30,7 @@
   "toolbar-button-upload-settings": "Open Upload Settings",
   "upload-notification": "Currently uploading",
   "upload-settings-modal-title": "Opencast Upload Settings",
-  "upload-settings-button-store": "Store Settings",
+  "upload-settings-button-store": "Apply settings",
```

Comment:
I still think "Save" would be the best word here, but I'm fine with this too.

Reply:
As long as it's not “store” → changed.
Though I think “Apply settings” is most common.